### ISSUE TITLE
Umbilical cord edema

### ISSUE THREAD

**Issue by poratshay** [obophenotype/human-phenotype-ontology#11570]
**Preferred term label:**
Umbilical cord edema

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0001-9947-035X

**Comment by pnrobinson** [comment]
@poratshay 
Adding new term: 
- ID: HP:6001475
- label: Umbilical cord edema
- parents: [Abnormality of the umbilical cord (HP:0010881)](https://hpo.jax.org/app/browse/term/HP:0010881)
- pmid: PMID:41133498|PMID:806296
- def: Visible edema (swelling) in a cord with a minimal cross sectional area of 1-3 cm2. This may result from an excessive accumulation of Wharton jelly, resulting in a thick or edematous umbilical cord.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11570
- synonyms:
